### ISSUE TITLE
Error during installation

### ISSUE THREAD

**Issue by vgopikaa** [EBI-Metagenomics/MGnifyR#36]
Hi, 
I am trying to install MGnifyR in my R desktop (version 4.3.3, bioconductor is up to date). I tried using BiocManager but failed to install the package. While installing from github using remotes(), I stumbled upon the following error. My understanding is that 4.3.3 is the latest and there isnt yet a ver 4.4.0 ?
Could you please help me out with this?

Thanks so much in advance
Gopikaa

![Screenshot 2024-03-07 093844](https://github.com/EBI-Metagenomics/MGnifyR/assets/162565896/bd953ccb-899f-49ba-9bcd-b8ddc55dff87)

**Comment by s-meaden** [comment]
I'm getting the same issue when installing from github but have installed MGnifyR on various machines in the past without this problem. Also getting the following error with the bioconductor install:

```
Warning message:
package ‘MGnifyR’ is not available for Bioconductor version '3.18'
```

Any help much appreciated!

**Comment by TuomasBorman** [comment]
Hello!

Sorry for very late response. I did not receive notification about this issue.

Related to Bioconductor submission, we had to use latest development version of R (which was 4.4.0). Now 4.4.0 is the release version of R, and MGnifyR package can be found from release version of Bioconductor so everything should be ok. If not, I'm happy to help. Now I get notifications on these issues so I will reply quickly.

-Tuomas
